catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookByIndexBrick.java

```diff
@@ -67,24 +66,12 @@ public int getViewResource() {
 	@Override
 	public View getView(Context context) {
 		super.getView(context);
-
```

Comment:
It is not necessary to override a method that just calls super and returns its result. You can just remove the entire method public View getView(Context context) {..}